crates/bitwarden-auth/Cargo.toml

```diff
@@ -17,20 +17,27 @@ keywords.workspace = true
 [features]
 wasm = [
     "bitwarden-core/wasm",
+    "bitwarden-crypto/wasm",
     "dep:tsify",
     "dep:wasm-bindgen",
     "dep:wasm-bindgen-futures"
 ] # WASM support
 
 # Note: dependencies must be alphabetized to pass the cargo sort check in the CI pipeline.
 [dependencies]
+bitwarden-api-api = { workspace = true }
 bitwarden-core = { workspace = true, features = ["internal"] }
-bitwarden-error = { workspace = true }
+bitwarden-crypto = { workspace = true }
+bitwarden-encoding = { workspace = true }
+bitwarden-error.workspace = true
```

Comment:
Nit❓ : why change this syntax from the original syntax which matches the others in the file?